### ISSUE TITLE
no-named-as-default

### ISSUE THREAD

**Issue by benmosher** [benmosher/eslint-plugin-import#23]
Need a better name.

Given: 
```js
// foo.js
export default 'foo';
export const bar = 'baz';
```

...this would be valid:
```js
import foo from './foo.js';
```

...and this would be reported:
```js
import bar from './foo.js';
```

Message something like `Using exported name 'bar' as identifier for default export.`

Rationale: using an exported name as the name of the default export is either 

- misleading: others familiar with `foo.js` probably expect the name to be `foo`
- a mistake: only needed to import `bar` and forgot the brackets (the case that is prompting this)
